lib/runtime.cpp

```diff
@@ -132,6 +132,9 @@ RuntimeSettings::RuntimeSettings(const std::string& prefix_, const std::string&
     }
 
     config = load_yaml(config_file);
+    if (config == nullptr) {
+        EVLOG_AND_THROW(EverestConfigError("Config file is nullptr!"));
+    }
```

Comment:
I think `load_yaml` always returns an nlohmann::json instance but it can be implicitely equal to a `nullptr`, in case it is an empty file.  So this is not an intrinsic error and the message is also wrong in this case.  If we're not able to deal with "empty" config files (which I think, could be valid) then we should throw a corresponding error like 'empty config files not supported'

Reply:
The problem doesn't seem to handle empty config files, but to handle broken config files. [This file](https://github.com/EVerest/everest-framework/blob/fix-unit-tests/tests/test_configs/broken_config_config.yaml) caused the error:
```
[json.exception.type_error.306] cannot use value() with null
```
I am also sure that the error occurs in [this line](https://github.com/EVerest/everest-framework/blob/9604e06820ea671b68d097870d2b80b6c99bfa1e/lib/runtime.cpp#L136)

Since there is a default value given it should'nt brake on empty configs (and it doesn't).
So it seems to make a difference laoding a broken config or an empty config